simtools/tests/test_array_model.py

```diff
@@ -1,13 +1,21 @@
 #!/usr/bin/python3
 
+import pytest
 import logging
 
 from simtools.model.array_model import ArrayModel
+from simtools.util.tests import (
+    has_db_connection,
+    has_config_file,
+    DB_CONNECTION_MSG,
+    CONFIG_FILE_MSG,
+)
 
 logger = logging.getLogger()
 logger.setLevel(logging.DEBUG)
 
 
+@pytest.mark.skipif(not has_config_file(), reason=CONFIG_FILE_MSG)
```

Comment:
I guess there is nothing significantly different between North and South that we want to run the tests for both sites.

Reply:
No, we don't. For the sake of unit tests, the specific site is not relevant.